Update API Components docs (#178)
mintlify/mint: docs/api-components/playground.mdx

---
title: "Playground"
description: "Enable users to interact with your API"
api: "GET https://server.mintlify.com/api/v1/demo"
hideApiMarker: true
---

<ParamField query="message" type="string" required>
  The message to be used in the query
</ParamField>

<ResponseExample>
  ```json Example Response
  {
    "success": true,
    "message": "Hello world"
  }
  ```
</ResponseExample>

## Introduction

> The API playground is an interactive environment to make requests and preview your API endpoints.

Reviewer: ```suggestion
The API playground is an interactive environment to make requests and preview an API endpoint.
```

To create an API page, you must set an `api` or `openapi` property in the page metadata. See [Configuration](api-components/configuration) for more details.

> ###

Reviewer: This might be a good place to mention other global api configurations too. Maybe have a title called `## Global API Configurations` and mention baseUrl and auth in addition to `hidePlayground`.

![image](https://user-images.githubusercontent.com/55263191/212524302-f4914142-cef3-4608-a471-f2fbad6b7a48.png)

Author: Do you think it should be in `/playground` or `/configuration`? Maybe `/configuration` makes more sense

Reviewer: Configuration makes sense in that it is technically configuration but the separation I see between the `/playground` and the `/configuration` page is that you put `hidePlayground` in `/playground` which makes me feel like all global configurations should go there? And the separation between the 2 pages is actually the granularity of the configuration? Either way we can also do this in another PR.

Author: Maybe in `/configuration` it can just be a link to the `/settings/global` page? I don't know if it's worth repeating the information, and it'll make it easier to extend with more config options in the future.

## Hiding API playgrounds

> You can hide API playgrounds site-wide by setting "hidePlayground" in your `mint.json` file.

Reviewer: ```suggestion
You can hide API playgrounds site-wide by setting `hidePlayground` in your `mint.json` file.
```

> ```json mint.json
> "api": {
>     "hidePlayground": true
> }
> ```

Reviewer: To show that it's a property in a larger json object.
```suggestion
```json mint.json
...
"api": {
    "hidePlayground": true
}
```
```

Author: @hahnbeelee This is inconsistent with the rest of the docs, examples [1](https://github.com/mintlify/mint/blob/569ab53681a1a9bcec10fc698ee49e9a610833eb/docs/settings/seo.mdx?plain=1#L27-L33), [2](https://github.com/mintlify/mint/blob/569ab53681a1a9bcec10fc698ee49e9a610833eb/docs/settings/navigation.mdx?plain=1#L16-L23), [3](https://github.com/mintlify/mint/blob/569ab53681a1a9bcec10fc698ee49e9a610833eb/docs/analytics/set-up/pirsch.mdx?plain=1#L5-L17). If we want to make this change let's make it in a separate PR.
